src/Pages/Home.jsx

```diff
@@ -2,45 +2,23 @@ import {useHistory} from 'react-router-dom';
     )
     const { isOpen, onOpen, onClose } = useDisclosure()
```

Comment:
change for signin `const { isOpen, onOpen, onClose } = useDisclosure() ` -> `{isOpen: isLoginModalOpen ...}`